### ISSUE TITLE
Add whole day events to the upcoming list

### ISSUE THREAD

**Issue by stan-voo** [deanishe/alfred-gcal#98]
Great workflow! Will you please add whole day events to the upcoming list?

**Comment by zzzgithubzzz** [comment]
Is there any work around to view 'All Day' events?

**Comment by jonnywilliamson** [comment]
Yes I was wondering why my upcoming events looked so sparse. I use all day events alot and they are appearing.

**Comment by zzzgithubzzz** [comment]
> Is there any work around to view 'All Day' events?

Any update on this? Can we get to see 'All Day' events in this awesome utility please..?

**Comment by zzzgithubzzz** [comment]
> Yes I was wondering why my upcoming events looked so sparse. I use all day events alot and they are appearing.

Please describe as to how you got them appearing...
